### DISCUSSION TITLE
browser compatibility issues with the useAnimate method

### DISCUSSION THREAD

**Discussion by Mrlilili** [vueuse/vueuse#3259] [General]
The following error appears on some earlier versions of the browser:
```
Uncaught DOMException: Failed to execute 'animate' on 'Element': Partial keyframes are not supported.
```
![image](https://github.com/vueuse/vueuse/assets/8361655/889f43d0-2de9-4b63-9959-2cf6a22eac71)

After querying, it was found that the attributes of each frame need to be the same, otherwise the above exception will occur.

@tolking  https://github.com/vueuse/vueuse/blob/main/packages/core/useAnimate/index.ts#L251

**Comment by tolking** [comment]
It seems that your browser does not support animate

**Reply by Mrlilili** [reply]
It's not a lack of support, but a problem with the input parameter structure.

**Reply by tolking** [reply]
It is related to [Implicit to/from keyframes](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Keyframe_Formats#implicit_tofrom_keyframes)
